### DISCUSSION TITLE
Absent Monster Cards

### DISCUSSION THREAD

**Discussion by TheSquinter** [Lurkars/gloomhavensecretariat#175] [Ideas]
I notice that not all monsters appear in the initial scenario setup view as they are set to appear when sections are revealed. I believe this info should be available to players to help in the initial preparation phase.

Curious if this is intended by design? XHA appears to show them. 

Perhaps have an option to "hide unrevealed enemies" if they have not appeared in the scenario yet for those that prefer not knowing?

**Comment by Lurkars** [comment]
I just added a scenario setup feature. When a scenario is selected, just click on the scenario name on the bottom bar. The used scenario standees should be visible as in the scenario book. When you check `Spoiler` the actual monster like with special rules etc also appear. (This will be fixed soon, that `Spoiler` button is only visible if there is something to spoil, so currently you may click it and nothing changes beside you can also click on the standee to see abilities and stats). When scenario is already running, there is a `Scenario Setup` checkbox on the scenario window to view the setup again.


For reference see #166.

**Comment by TheSquinter** [comment]
Hmm not sure if that's helping. For example, if we check a scenario like 19 which is missing the spawn rules currently, no monsters are shown at all. Having all the monsters available would help situations like this where the section or scenario rule data isn't complete yet.

p.s. I tried using Sc 14 as a template to help populate the json for Scenario 19 if you need it. Syntax is probably wrong, especially the 'or' operators and '},' in the wrong locations.

Edit: I created a new pull request with the Scenario 19 rules tested and implemented
